Branch1

Mohamed-js opened the pull request:
The first Microverse Collaborative Project - after the first review.

The Newyork Times article page like.

We used in this project :
-HTML
-CSS

We divided the page into some parts like " navbar - heading - article - outline images - inline images - the articles grid - footer "

Then we styled them gave them some styles and positioned them using Flex, Float and Grid systems.

We are waiting for your comments in the review.

Mohamed-js committed:
Set up project and make navbar

I have added images, linters and html files and started working on navbar in html.

charlyeneh committed:
Add some parts of the project body

Mohamed-js committed:
Finish Footer and Style some elements.

charlyeneh committed:
Complete the remaining styling

Srjnnnn reviewed (changes requested):
## Status: Changes Required :recycle: :stop_sign: 
Hello team! It's a wonderful job. You've made all the mandatory requirements. :muscle: Your semantic HTML usage, understanding of the positioning are just great. However, before moving on, let's make the site looks more identical to the original page. It'd be good practice for you before the Capstone project. :100: 

- [x] Please add the 'More in space' and 'Editor's pick' sections down at your page.

- [ ] Please add the necessary links at the footer at most down part of the page.

- [ ] Please decrease the gap between News, Opinion, Arts, and Living sections. 

- [ ] Please change the color of the links that are clickable from blue to black.

- [ ] Please try to mimic the Editor's Pick section on the right side. 

- [ ] You can improve your Readme. You can change those githubhandles and add a screenshot of your page.

Again it looks amazing. If you do these requested changes, it'll even get better and will look like exactly the original page. When you are done with your changes, please submit for a re-review. If you have any questions about the review, feel free to reach me via Slack :100: 
Eyüp Sercan UYGUR

Mohamed-js committed:
Edit review requirement

-Add "More in space " and "Editors' pick" section.

-Add all footer links.

-Decrease gaps in footer.

-Changed links' colors to black.

-Edit "Editors' pick" section.
